Documentation/ReviewingGuidelines.txt

```diff
@@ -0,0 +1,116 @@
+  diff.
+
+- If a patch is long, you can delete irrelevant chunks of the patch from your
+  reply. Make sure you leave enough to provide the appropriate context for your
+  comments!
+
+- When pointing out an issue, try to include suggestions for how the author
+  could fix it. This not only helps the author to understand and fix the issue,
+  it also deepens and improves your understanding of the topic.
+
+- Reviews do not need to exclusively point out problems. Feel free to "think out
+  loud" in your review: describe how you read & understood a complex section of
+  a patch, ask a question about something that confused you, point out something
+  you found particularly well-written, etc.
+
+- When providing a recommendation, be as clear as possible about whether you
+  consider it "blocking" (the code would be broken or otherwise made worse if an
+  issue isn't fixed) or "non-blocking" (the patch could be made better by taking
+  the recommendation, but acceptance of the series does not require it).
+  Non-blocking recommendations can be particularly ambiguous when they are
+  related to - but outside the scope of - a series ("nice-to-have"s), or when
+  they represent only stylistic differences between the author and reviewer.
+
+- If you cannot complete a full review of a series all at once, consider letting
+  the author know (on- or off-list) if/when you plan to review the rest of the
+  series.
+
+- If you read and review a series but find nothing that warrants inline
+  commentary, reply to the series' cover letter to indicate that you've reviewed
+  the changes.
+
+Completing a review
+~~~~~~~~~~~~~~~~~~~
+Once each patch of a series is reviewed, the author (and/or other contributors)
+may discuss the review(s). This may result in no changes being applied, or the
+author will send a new version of their patch(es).
+
+After a series is rerolled in response to your or others' review, make sure to
+re-review the updates. If you are happy with the state of the patch series,
+explicitly indicate your approval (typically with a reply to the latest
+version's cover letter). Optionally, you can let the author know that they can
+add a "Reviewed-by: <you>" trailer to subsequent versions of their series.
+
+Terminology
+-----------
+nit: ::
+	Denotes a small issue that should be fixed, such as a typographical
+	error or mis-alignment of conditions in an `if()` statement.
+
+aside: ::
+optional: ::
+non-blocking: ::
+	Indicates to the reader that the following comment should not block the
+	acceptance of the patch or series. These are typically recommendations
+	related to code organization & style, or musings about topics related to
+	the patch in question, but beyond its scope.
+
+s/<before>/<after>::
+	Shorthand for "you wrote <before>, but I think you meant <after>,"
+	usually for misspellings or other typographical errors. The syntax is a
+	reference to the `vi` "Substitute" command.
```

Comment:
I believe that the `s/<regex>/<replacement>/` syntax originates from the [`ed`](https://en.wikipedia.org/wiki/Ed_(text_editor)) command and is "only" imitated by [`sed`](https://en.wikipedia.org/wiki/Sed), [`ex`](https://en.wikipedia.org/wiki/Ex_(text_editor)) (which `vi` uses when you enter the `:` mode), `vim` and `perl`.

Maybe reword it to something like "The syntax refers to the "substitute" command known from Unix tools like `ed`, `sed`, `vim` and `perl`, to silence nit-pickers such as myself?

Reply:
My nit is that it should be `s/<before>/after/`, I think. ;)